src/Assets/scss/custom/_request.scss

```diff
@@ -80,4 +89,144 @@
 .like-card:hover {
   cursor: pointer;
   font-size: 23px !important;
+}
+
+.chat-icon {
+  position: fixed;
+  bottom: 0.5vw;
+  right: 0.5vw;
+  width: 50px;
+  font-size: 30px;
+  box-shadow: -5px 5px 5px 1px rgba(0,0,0,0.60);
+  z-index: 5000;
+}
+.chat-icon:hover {
+  cursor: pointer;
+}
+
+.profile{
+  margin: 0 auto;
+  text-align: center;
+}
+
+.img-circle {
+  width: 60px;
+  height: 60px;
+  border-radius: 100%;
+}
+
+.label-picture {
+  margin-top: -10px;
+  text-align: center;
+}
+
+.avatar-sm, .font-avatar {
+  width: 40px;
+  height: 40px;
+  font-size: xx-large;
+  border-radius: 50%;
+}
+
+.user-model:hover {
+  background-color: grey;
+  cursor: pointer;
+  color: white;
+}
+.user-model {
+  color: grey;
+}
+.reciever {
+  padding: 1vw 1vw 1vw 1vw;
+  margin: 1vw 50% 1vw 1vw;
+  background-color: #3361FF;
+  border-radius: .4em;
+  color: white;
+  position: relative;
+}
+.author {
+  position: relative;
+  padding: 1vw 1vw 1vw 1vw;
```

Comment:
great job @urban the chat looks nice fresh and clean but always use a simpler approach in CSS this `padding: 1vw 1vw 1vw 1vw;` is the same as `padding: 1vw`; on line `148` AND `139` otherwise  :+1: :+1:

Reply:
Thanks Mr. Designer 🤗